src/ucp/core/ucp_rkey.c

```diff
@@ -34,6 +34,26 @@ size_t ucp_rkey_packed_size(ucp_context_h context, ucp_md_map_t md_map)
     return size;
 }
 
+void ucp_rkey_packed_copy(ucp_context_h context, ucp_md_map_t md_map,
+                          void *rkey_buffer, const void* uct_rkeys[])
```

Comment:
const void* uct_rkeys[] is interpreted as void**, as I can see incoming parameter is just non-initialized array (and coverity agree with me :) )

Reply:
it should be a const array of pointers

Reply:
yes, I see
the simplest way to suppress error (without hack with suppress coverity by comment) is create separate array of pointers, like
```
const void *array[2] = {key_buf, NULL};
```
and pass it to this function (technically compiler makes same things, should not affect performane)